packages/fx-core/src/common/versionMetadata.ts

```diff
@@ -18,6 +26,13 @@ export const MetadataV3 = {
   teamsManifestFileName: "manifest.json",
   aadManifestFileName: "aad.manifest.json",
   v3UpgradeWikiLink: "https://aka.ms/teams-toolkit-5.0-upgrade",
+  secretFileComment:
+    "# This file includes environment variables that will not be committed to git by default. You can set these environment variables in your CI/CD system for your project." +
+    EOL,
+  secretComment:
+    "# Secrets. Keys prefixed with `SECRET_` will be masked in Teams Toolkit logs." + EOL,
+  envFileComment:
```

Comment:
.env.local has different comment than .env.dev, see https://github.com/OfficeDev/TeamsFx/blob/dev/templates/scenarios/js/sso-tab/env/.env.local.tpl#L1

Reply:
got it